matrix_sdk_base/src/store/memory_store.rs

```diff
@@ -303,6 +303,14 @@ impl MemoryStore {
         #[allow(clippy::map_clone)]
         self.stripped_room_info.iter().map(|r| r.clone()).collect()
     }
+
+    async fn get_account_data_event(&self, event_type: EventType) -> Result<Option<AnyBasicEvent>> {
+        #[allow(clippy::map_clone)]
```

Comment:
Does `Option::cloned` not work here?

Reply:
No, I get this:

> the following trait bounds were not satisfied:
> `std::option::Option<dashmap::mapref::one::Ref<'_, std::string::String, AnyBasicEvent>>: Iterator`

However I tested and it doesn't need the `allow` attribute, I just copied it from the previous function without thinking.